test/template-tests/testcases/deploymentTemplate/Location-Should-Not-Be-Hardcoded.test.ps1

```diff
@@ -26,7 +26,7 @@ if ($locationParameter -and
     "$($locationParameter.defaultValue)".Trim() -ne 'global' -and 
     $IsMainTemplate) {
     # If it wasn't, write an error
-    Write-Error "Location parameter must not be hardcoded.  The default value should be [resourceGroup().location]." -ErrorId Location.Parameter.Hardcoded -TargetObject $parameter
+    Write-Error "Location parameter must not be hardcoded.  The default value should be [resourceGroup().location]. It is $($locationParameter.defaultValue)" -ErrorId Location.Parameter.Hardcoded -TargetObject $parameter

```

Comment:
Change message:
The defaultValue of the location parameter must not be a specific location.  The defaultValue must be... (the rest is fine)